pkg/webhooks/hiveextension/v1beta1/agentclusterinstall_validation_hook.go

```diff
@@ -397,9 +398,9 @@ func isSNO(newObject *hiveext.AgentClusterInstall) bool {
 		newObject.Spec.ProvisionRequirements.WorkerAgents == 0
 }
 
-func getHighAvailabilityMode(originalObject, updatesObject *hiveext.AgentClusterInstall) *string {
```

Comment:
Don't change this function, in k8s API the control plane count is given by default

Reply:
You can look at `Spec.ProvisionRequirements.ControlPlaneAgents`